Automattic/wp-calypso
# Allows for dismissing the time zone mismatch notice

#### Changes proposed in this Pull Request

* Adds preferences to dismiss the time mismatch notice on a per-site basis.

#### Testing instructions



1. Create a Jetpack site with a paid plan.
2. Visit the AL, Backup, or Scan pages.
3. Observe time mismatch notice.
4. Dismiss the notice.
5. Check AL, Backup, and Scan pages for the notice, before and after a refresh.

Fixes 1164141197617539-as-1199087653467461.
